duckduckgo/Android
# Update widget design

Task/Issue URL: https://app.asana.com/0/361428290920652/1121098489843338
Tech Design URL: N/A

**Description**:
Update widget to match latest design

**Steps to test this PR**:
New installation Test:
1. Add the widgets on a range of devices and ensure they look sensible
1. Also try alternative launchers such as Lawnchair where we previously had an issue, see https://github.com/duckduckgo/Android/issues/166

Upgrade Test:
1. Install the current version of the app and add both the dark and light widget to your home scree
1. Install this version over the top
1. Ensure the widgets are updated

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
